lib/Serialization/ModuleFormat.h

```diff
@@ -1099,7 +1099,8 @@ namespace decls_block {
     ValueOwnershipField, // inout, shared or owned?
     BCFixed<1>,          // isolated
     BCFixed<1>,          // noDerivative?
-    BCFixed<1>           // compileTimeConst
+    BCFixed<1>,          // compileTimeConst
+    BCFixed<1>           // distributedKnownLocal
```

Comment:
Please bump the module format version when you make changes.

Reply:
bumped